.github/assets/check_wasm.sh

```diff
@@ -74,6 +74,9 @@ exclude_crates=(
   reth-scroll-cli # tokio
   reth-scroll-node # tokio
   reth-scroll # tokio
+  reth-scroll-state-commitment # tokio
+  reth-scroll-chainspec # tokio
+  reth-scroll-evm # tokio
```

Comment:
It would be good if we could maintain `no_std` / `wasm` for these crates as we will need them in the derivation pipeline / stateless block verifier. Can you create an issue so that we can track that we need to make required crates `no_std` compatible for this ambition please?

Reply:
https://github.com/scroll-tech/reth/issues/108